numpyro/optim.py

```diff
@@ -75,6 +77,27 @@ def __init__(self, *args, **kwargs):
         super(Adam, self).__init__(optimizers.adam, *args, **kwargs)
 
 
+class ClippedAdam(_NumpyroOptim):
```

Comment:
Note Martin's `ClippedAdam` also includes learning rate decay. We don't need to implement that now, but can you make sure this errors if an `lrd` kwarg is found?

Reply:
I think we can support lrd too, because we can provide a schedule for the learning rate to the jax optimizer. I will try to include that or error if it turns out to be more complicated.

Reply:
Done.